Gemfile

```diff
@@ -5,6 +5,7 @@ gemspec
 if RUBY_VERSION < '2.2'
   gem 'sinatra', '< 2'
   gem 'rack', '>= 1.1', '< 2.0.0'
+  gem 'parallel', '< 1.14'
```

Comment:
It's probably better to pin the dependency:

```ruby
  if RUBY_VERSION < '2.1'
    gem 'rubocop-checkstyle_formatter', '~> 0.3.0'
  else
    gem 'rubocop-checkstyle_formatter'
  end
```

Reply:
This might or might not work implicitly, but `rubocop-checkstyle_formatter` itself is not the problem, but `rubocop` with it's pin of `parallel` not being strict enough.

Reply:
I took this ouf of the Gemfile now, no idea why it didn't work only in test.rb at the first try :man_shrugging: